### ISSUE TITLE
FTS and FTS-UI services fail to restart after reboot with FTS2.1 on Raspberry Pi

### ISSUE THREAD

**Issue by ki6uve** [FreeTAKTeam/FreeTakServer#688]
After fresh installation of FTS 2.1 on a raspberry pi computer, the FTS and FTS-UI services consistently fail to restart.  Services are confirmed to be set to enabled.  

[fts-stderr.log](https://github.com/FreeTAKTeam/FreeTakServer/files/14796667/fts-stderr.log)

**Comment by brothercorvo** [comment]
from your log, it appears that you have 2 set of services running. Ensure that you are running ZeroTpuch on a fresh installation of Ubuntu

**Comment by ki6uve** [comment]
Yes, it was a fresh install with a newly flashed SD card.  Used the ZTI for the build.

**Comment by ki6uve** [comment]
After further testing, we were able to confirm that the services were indeed being started after reboot but are not registering their "started" status with the system and therefore showing incorrect status with 'systemctl status fts' command.

**Comment by naman108** [comment]
this seems to be an issue with the latest releases, that services aren't being registered as started when, in fact, they are
